javascript/javascript1/week2/event_application.js

```diff
@@ -0,0 +1,11 @@
+
+function getEventWeekDay(num){
+
+    const todaysDate = new Date();
+    const dayName = todaysDate.toLocaleString('en-US', {weekday: 'long'});
```

Comment:
Nice and simple solution; excellent! 💪 Tiniest nitpick: is `dayName` needed? 😄

Reply:
not really 😃 
i was googling and try to find out how to get the weekday
they were different options.